commerce-theme-minium/commerce-theme-minium-impl/src/main/java/com/liferay/commerce/theme/minium/internal/product/renderer/list/entry/MiniumCPContentListEntryRenderer.java

```diff
@@ -273,11 +273,11 @@ public void render(
 		Set<String> dependencies = new HashSet<>();
 
 		dependencies.add(
-			"commerce-frontend-taglib@2.1.6/add_to_cart/AddToCartButton.es");
+			"commerce-frontend-taglib@2.1.7/add_to_cart/AddToCartButton.es");
 		dependencies.add(
-			"commerce-frontend-taglib@2.1.6/compare_checkbox" +
+			"commerce-frontend-taglib@2.1.7/compare_checkbox" +
 				"/CompareCheckbox.es");
-		dependencies.add("commerce-frontend-taglib@2.1.6/price/Price.es");
+		dependencies.add("commerce-frontend-taglib@2.1.7/price/Price.es");
 
```

Comment:
These changes reflect the frontend taglib version update (https://github.com/liferay/com-liferay-commerce/blob/7.1.x-next/commerce-frontend-taglib/bnd.bnd#L3)